### PR TITLE
Created Avatar Component

### PR DESCRIPTION
**Pull request by Sharmaa-Ansh**
Avatar Component has been created a;png with the page structure.

### CODE REVIEW

File: Components/Avatar/Avatar.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+height: 6.25rem;
+width: 6.25rem;
+}
+
+.Avatar__Img-lg{
+    height: 9.375rem;
+    width: 9.375rem;
+}    
+
+.Avatar-Standard{
```
Comment (shrey27) [inline Components/Avatar/Avatar.css:25]: try to be consistent with class names. Like you can go with 
Avatar__Image-sm, 
Avatar__Image-md,
Avatar__Image-lg,
Avatar__Image-sd

**Review by shrey27 — APPROVED** [review]

File: Components/Avatar/Avatar.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                    alt="">
```
Comment (anuj-kumary) [inline Components/Avatar/Avatar.html:81]: Always give some value to **alt**.

Comment (Sharmaa-Ansh) [reply Components/Avatar/Avatar.html:81]: noted

File: Components/Avatar/Avatar.css (hunk 1 above)
Comment (Sharmaa-Ansh) [reply Components/Avatar/Avatar.css:25]: noted

**Pull request merged**